.env

```diff
@@ -0,0 +1,7 @@
+PORT = "4000"
+DB_USER = "root"
+DB_HOST = "127.0.0.1"
+DB_PORT = "3306"
+DB_DIALECT = "mysql"
+DB_NAME = "planIt"
+ACCESS_TOKEN_SECRET = "holaaaa"
```

Comment:
Let's not track the.env file in git and let's also add the missing variables to the .env.example file